connection/src/thick.rs

```diff
@@ -52,6 +53,7 @@ use std::{
     ops::Range,
     result::Result as StdResult,
     sync::Arc,
+    time::{SystemTime, UNIX_EPOCH},
```

Comment:
I just recently ran across , [mc_common::time::SystemTImeProvider](https://github.com/mobilecoinfoundation/mobilecoin/blob/31eb5a0c92d8f12fca11abe6049d839e826c1d13/common/src/time.rs#L26)
Probably should be using that here.

Reply:
Good call. Replaced `SystemTime` code with `SystemTimeProvider`